Dockerfile

```diff
@@ -8,51 +8,33 @@
 #
 # See the `deployments/README.md` for all the available `PROJECT` values.
 
-# This first stage prepares our dependencies to be built by `cargo-chef`.
-FROM rust as planner
+FROM docker.io/paritytech/bridge-dependencies as builder
```

Comment:
Is it not good enough to just do `paritytech/bridge-dependencies`?

Reply:
it's better to have the full names when building with something that's not docker, we use buildah.